radio/src/storage/sdcard_yaml.cpp

```diff
@@ -92,15 +133,43 @@ void storageCreateModelsList()
 // SDCARD storage interface
 //
 
+static const char * attemptLoad(const char *filename, uint16_t* checksum_status)
```

Comment:
Same here as before, I would prefer references for checksum_status.

Reply:
Yeah, but I also want to have an easy way of controlling wether to not readYamlFile() should do the checksum calculation. Currently the pointer to write the result is controlling this.

I will update the PR - hopefully my intention is a bit clearer now. If you have an idea to do this simpler please comment :-)

Reply:
Yes you are right, as along as the same function is used for files with and without checksum, this is probably the best way to do it.